ND Supporting Document 2_2.adoc

```diff
@@ -2126,8 +2126,7 @@ The evaluator shall verify that the Finished message (Content type hexadecimal 1
 . Test 1 [conditional]: If ECDHE ciphersuites/groups are supported:
 [loweralpha]
 .. The evaluator shall repeat this test for each supported elliptic curve. The evaluator shall attempt a connection using a supported ECDHE ciphersuite (DTLS 1.0/1.2) or group (DTLS 1.3) and a single supported elliptic curve specified in the supported groups extension. The evaluator shall verify (through a packet capture or instrumented client) that the TOE selects the same curve in the Server Key Exchange (DTLS 1.0/1.2) or Server Hello (key_share, for DTLS 1.3) message and successfully establishes the connection.
-
-.. The evaluator shall attempt a connection using a supported ECDHE ciphersuite group (DTLS 1.3) and a single unsupported elliptic curve (e.g. secp192r1 (0x13)) specified in RFC4492, chap. 5.1.1. The evaluator shall verify that the TOE does not send a Server Hello message and the connection is not successfully established.
+.. The evaluator shall attempt a connection using a supported ECDHE ciphersuite (DTLS 1.0/1.2) or group (DTLS 1.3) and a single unsupported elliptic curve (e.g. secp192r1 (0x13)) specified in RFC4492, chap. 5.1.1. The evaluator shall verify that the TOE does not send a Server Hello message and the connection is not successfully established.
```

Comment:
@ouarak Thank you for pointing this out. I'll open an issue on this so that we can discuss as a team on on call.

Reply:
Based on the discussion today, I think the original "ECDHE ciphersuite group" and proposed update "supported ECDHE...group (DTLS 1.3) and a single unsupported elliptic curve" use unclear terminology.

I propose: The evaluator shall attempt a connection using a supported_groups extension containing a single unsupported elliptic curve (e.g. secp192r1 (0x13)). Note: For DTLS 1.2 connections, a single supported ECDHE ciphersuite shall be proposed. The evaluator shall verify that the TOE does not send a Server Hello message and the connection is not successfully established.

Both 1.2 and 1.3 use the supported_groups extension. Since this is an ECDHE test, the supported_groups extension needs to specify an elliptic curve, not a finite field. 1.3 does not include key agreement in the ciphersuite, so using an ECDHE ciphersuite is only relevant to 1.2.